source/_cookbook/dim_and_brighten_lights.markdown

```diff
@@ -0,0 +1,164 @@
+---
+layout: page
+title: "Dim (and brighten) lights via a remote"
+description: "The scripts and automations to allow you to use a remote to dim and brighten a light"
+date: 2017-09-06 18:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Automation Examples
+---
+
+This requires both a light on a dimmer, and a z-wave remote control that sends one scene when a button is held, and another when released. This ensures that the scripts (which follow) are stopped, avoiding the risks of a script that never ends.
```

Comment:
I am working on implementing a set of standards for our documentation as to provide consistency and reduce confusion, while also improving a professional appearance to our users. Part of this will include proper casing of brands, names, protocols, services, etc.

- Change "z-wave" -> "Z-Wave"
- Change "a light on a dimmer" -> "a dimmable light"